rebench/tests/executor_test.py

```diff
@@ -57,6 +58,30 @@ def test_broken_command_format_with_ValueError(self):
             ex.execute()
         self.assertIsInstance(err.exception.source_exception, ValueError)
 
+    def test_remove_executors_with_missing_binary(self):
+        options = ReBench().shell_options().parse_args(['dummy'])
+        cnf = Configurator(load_config(self._path + '/test.conf'), DataStore(self.ui),
```

Comment:
I think we would want to test this for the three different schedulers.
If I remember correctly, the Configurator takes a scheduler as parameter?